Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3169,6 +3169,45 @@ public void onActivityDestroyed(Activity activity) {
 
     /**
      * <p>An Interface class that is implemented by all classes that make use of
+     * {@link BranchLinkShareListener}, defining methods to listen for link sharing status.</p>
+     */
+    public interface BranchLinkShareListener2 {
```

Comment:
Instead of just duplicating the functionality, not sure, but is there a way to just extend the previous. Also, name change please.